docs/usage/self-hosted-configuration.md

```diff
@@ -628,6 +628,25 @@ JSON files will be stored inside the `cacheDir` beside the existing file-based p
 
 ## repositoryCacheType
 
+Set this to an S3 URI to enable S3 backed repository cache.
+
+```ts
+{
+  repositoryCacheType: 's3://bucket-name';
+}
+```
+
+<!-- prettier-ignore -->
+!!! note
+    The following environment variables should be set in order for the S3 client to work.
```

Comment:
add a note, that ECS IAM is also supported, so no extra env is required

https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/loading-node-credentials-iam.html

Reply:
the link i posted is v3, i guess ill add v3 docs ver
https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/loading-node-credentials-iam.html